README.md

```diff
@@ -260,7 +260,19 @@ show all upstreams.
 Consul_interface
 ======
 
-you can add or delete backend server through consul_ui or http_interface.
+Data can be taken from key/value store or service catalog. In the first case parameter upsync_type of directive must be *consul*. For example
+
+```nginx-consul
+        upsync 127.0.0.1:8500/v1/kv/upstreams/test upsync_timeout=6m upsync_interval=500ms upsync_type=consul strong_dependency=off;
+```
+
+In the second case it must be *consul_services*.
+
+```nginx-consul
+        upsync 127.0.0.1:8500/v1/catalog/service/test upsync_timeout=6m upsync_interval=500ms upsync_type=consul_services strong_dependency=off;
+```
+
+You can add or delete backend server through consul_ui or http_interface. Below are examples for key/value store.
 
 http_interface example:
```

Comment:
Here are examples about addition and deletion KV. If I understand this well, catalogue of services is mostly about static configuration in consul configs. Additionally, I do not know consul interface very well, so at this moment I do not know what examples should be added here.